openlibrary/core/vendors.py

```diff
@@ -167,7 +246,32 @@ def cached_get_amazon_metadata(*args, **kwargs):
         result = memoized_get_amazon_metadata.update(*args, **kwargs)[0]
     return result
 
+
+@public
+def get_betterworldbooks_metadata(isbn):
+    """
+    :param str isbn: Unormalisied ISBN10 or ISBN13
+    :return: Metadata for a single BWB book, currently listed on their catalog, or error dict.
+    :rtype: dict
+    """
+
+    isbn = normalize_isbn(isbn)
+    try:
+        if isbn:
+            return _get_betterworldbooks_metadata(isbn)
+    except Exception:
+        return {}
```

Comment:
The return values here need testing, it looks like `None` is a possible return value (because `normalize_isbn()` _can_ return `None` if the value is not isbn like at all) , but I'm not sure `{}` can ever be returned as exceptions are handled in `_get_betterworldbooks_metadata()` and meaningful error json is returned. Possibly for another PR, this PR is meant to be Amazon focused, but I filled in docstrings for everything while I was on a roll.

Reply:
> but I filled in docstrings for everything while I was on a roll

😍 

> Possibly for another PR, this PR is meant to be Amazon focused

That seems reasonable to me.